Source/Core/DiscIO/DiscUtils.h

```diff
@@ -49,24 +50,32 @@ constexpr u32 WII_NONPARTITION_DISCHEADER_SIZE = 0x100;
 constexpr u32 WII_REGION_DATA_ADDRESS = 0x4E000;
 constexpr u32 WII_REGION_DATA_SIZE = 0x20;
 
-// Default block size for GCZ/WIA/RVZ images
-constexpr int PREFERRED_BLOCK_SIZE = 0x20000;  // 128 KiB
+// 128 KiB (0x20000) is the default block size for GCZ/RVZ images
+constexpr int GCZ_RVZ_PREFERRED_BLOCK_SIZE = 0x20000;
 
-// Because DVD timings are emulated as if we can't read less than an entire ECC block at once
-// (32 KiB - 0x8000), there is little reason to use a block size smaller than that.
-constexpr int PREFERRED_MIN_BLOCK_SIZE = 0x8000;  // 32 KiB
+// 32 KiB (0x8000) was picked because DVD timings are emulated as if we can't read less than
+// an entire ECC block at once. Therefore, little reason to choose a smaller block size.
+constexpr int PREFERRED_MIN_BLOCK_SIZE = 0x8000;
 
-// For performance reasons, blocks shouldn't be too large.
 // 2 MiB (0x200000) was picked because it is the smallest block size supported by WIA.
-constexpr int PREFERRED_MAX_BLOCK_SIZE = 0x200000;  // 2 MiB
+// For performance reasons, blocks shouldn't be too large.
+constexpr int PREFERRED_MAX_BLOCK_SIZE = 0x200000;
 
+// 16 KiB (0x4000) for supporting GCZs in versions of Dolphin prior to 5.0-11893
```

Comment:
I think it can be a little misleading to state this before you get to the "if we didn't find a good block size" qualifier. Most of the time it's fine to use at least 32 KiB when you want compatibility with older versions of Dolphin.